eng/Versions.props

```diff
@@ -34,9 +34,9 @@
     <!-- emsdk -->
     <MicrosoftNETWorkloadEmscriptenPackageVersion>8.0.0</MicrosoftNETWorkloadEmscriptenPackageVersion>
     <!-- wasdk -->
-    <MicrosoftWindowsAppSDKPackageVersion>1.3.230724000</MicrosoftWindowsAppSDKPackageVersion>
+    <MicrosoftWindowsAppSDKPackageVersion>1.5.240205001-preview1</MicrosoftWindowsAppSDKPackageVersion>
     <MicrosoftWindowsSDKBuildToolsPackageVersion>10.0.22621.756</MicrosoftWindowsSDKBuildToolsPackageVersion>
-    <MicrosoftGraphicsWin2DPackageVersion>1.0.5.1</MicrosoftGraphicsWin2DPackageVersion>
+    <MicrosoftGraphicsWin2DPackageVersion>1.1.1</MicrosoftGraphicsWin2DPackageVersion>
```

Comment:
```
D:\a\_work\1\s\src\Graphics\src\Graphics.Win2D\Graphics.Win2D.csproj : error NU1701: Warning As Error: Package 'Microsoft.Graphics.Win2D 1.1.1' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8, .NETFramework,Version=v4.8.1' instead of the project target framework 'net7.0-windows10.0.19041'. This package may not be fully compatible with your project. [D:\a\_work\1\s\Microsoft.Maui.BuildTasks.slnf]
D:\a\_work\1\s\src\Graphics\src\Graphics\Graphics.csproj : error NU1701: Warning As Error: Package 'Microsoft.Graphics.Win2D 1.1.1' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8, .NETFramework,Version=v4.8.1' instead of the project target framework 'net7.0-windows10.0.19041'. This package may not be fully compatible with your project. [D:\a\_work\1\s\Microsoft.Maui.BuildTasks.slnf]
D:\a\_work\1\s\src\Essentials\src\Essentials.csproj : error NU1701: Warning As Error: Package 'Microsoft.Graphics.Win2D 1.1.1' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8, .NETFramework,Version=v4.8.1' instead of the project target framework 'net7.0-windows10.0.19041'. This package may not be fully compatible with your project. [D:\a\_work\1\s\Microsoft.Maui.BuildTasks.slnf]
D:\a\_work\1\s\src\Core\src\Core.csproj : error NU1701: Warning As Error: Package 'Microsoft.Graphics.Win2D 1.1.1' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8, .NETFramework,Version=v4.8.1' instead of the project target framework 'net8.0-windows10.0.19041'. This package may not be fully compatible with your project. [D:\a\_work\1\s\Microsoft.Maui.BuildTasks.slnf]
D:\a\_work\1\s\src\Controls\src\Core\Controls.Core.csproj : error NU1701: Warning As Error: Package 'Microsoft.Graphics.Win2D 1.1.1' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8, .NETFramework,Version=v4.8.1' instead of the project target framework 'net8.0-windows10.0.19041'. This package may not be fully compatible with your project. [D:\a\_work\1\s\Microsoft.Maui.BuildTasks.slnf]
D:\a\_work\1\s\src\Controls\src\Xaml\Controls.Xaml.csproj : error NU1701: Warning As Error: Package 'Microsoft.Graphics.Win2D 1.1.1' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8, .NETFramework,Version=v4.8.1' instead of the project target framework 'net8.0-windows10.0.19041'. This package may not be fully compatible with your project. [D:\a\_work\1\s\Microsoft.Maui.BuildTasks.slnf]
D:\a\_work\1\s\src\Graphics\src\Graphics.Skia\Graphics.Skia.csproj : error NU1701: Warning As Error: Package 'Microsoft.Graphics.Win2D 1.1.1' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8, .NETFramework,Version=v4.8.1' instead of the project target framework 'net7.0-windows10.0.19041'. This package may not be fully compatible with your project. [D:\a\_work\1\s\Microsoft.Maui.BuildTasks.slnf]
```